src/Tracking.php

```diff
@@ -243,7 +244,7 @@ public static function hook_add_to_cart_event( $cart_item_key, $product_id, $qua
 		$object_id = empty( $variation_id ) ? $product_id : $variation_id;
 		$product   = wc_get_product( $object_id );
 
-		$product_price = WC()->cart->display_prices_including_tax() ? wc_get_price_including_tax( $product ) : wc_get_price_excluding_tax( $product );
+		$product_price = self::get_product_price_with_taxes( $product );
```

Comment:
I suggest renaming because the method can return both prices with and w/o taxes.

```suggestion
		$product_price = self::get_product_price( $product );
```